OpenOversight/app/models/emails.py

```diff
@@ -39,6 +39,19 @@ def __init__(self, receiver: str, user, token: str):
         super().__init__(body, subject, receiver)
 
 
+class ChangePasswordEmail(Email):
+    def __init__(self, receiver: str, user):
+        subject = (
+            f"{current_app.config['OO_MAIL_SUBJECT_PREFIX']} Your Password has Changed"
```

Comment:
Nit for consistency
```suggestion
            f"{current_app.config['OO_MAIL_SUBJECT_PREFIX']} Your Password Has Changed"
```

Reply:
Good call! I thought title case was having the `h` be lower case, but that is not correct. I'll make the change now!